index.html

```diff
@@ -310,6 +310,16 @@ <h3>Authorization</h3>
 or legacy authorization technologies.
       </p>
 
+      <section>
+        <h4>Forbidden Authorization</h4>
+        <p>
+Any authorization protocol that uses long-lived static credentials such as
+usernames and passwords or similar values in a request to the VC HTTP API MUST
+NOT be utilized. An example of such a forbidden protocol is HTTP Basic
+Authentication [[RFC7617]].
```

Comment:
```suggestion
Requests to the VC HTTP API MUST NOT utilize any authorization protocol that
uses long-lived static credentials such as usernames and passwords or similar
values. An example of such a forbidden protocol is HTTP Basic Authentication
[[RFC7617]].
```

Reply:
This suggestion changes the meaning of the normative statement. We don't want people putting usernames and passwords in HTTP Requests as a part of the authorization process when calling the HTTP API. We are fine w/ people using usernames and passwords to get an authorization token in separate process.

Please update the rewording to capture that critical difference.